templates/es6/www

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/**
+ * Entrypoint to application for es6 support
+ */
+ 
+require("babel-core/register");
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app').default;
```

Comment:
What I was seeing the other night was that `require("babel-core/register");` was working as expected for all files *outside* the app entrypoint, i.e. this file, but didn't seem to affect the actual entrypoint. I'll look into this some more though to get a better idea.